### DISCUSSION TITLE
Side-by-side stereoscopic 3D support

### DISCUSSION THREAD

**Discussion by ChrisFratz** [open-goal/jak-project#3568] [Ideas]
Would it be possible to add or recreate the proper 2-camera 3D support that was added to these games on the PS3 versions? I saw someone post a guide of some kind in the mod section on the discord for using reshade to accomplish this, and while it's good at faking 3D, it can never come close to the quality of a native implementation. I've been trying to play through the PS3 version of the game on RPCS3 but my computer isn't strong enough to maintain full speed and the 3D mode in that version locks the game to 30 FPS, So having it added as an option to OpenGOAL would be really awesome. Below is a screenshot of the 3D mode of the PS3 version running on RPCS3.
![image](https://github.com/open-goal/jak-project/assets/28227666/27fbf5b9-0d52-4e2d-a0f1-b23111d1fbe6)

**Comment by leansteak096-blip** [comment]
I totally agree. It enhances the experience.
